Components/NavigationFooter/_style.scss

```diff
@@ -1,6 +1,24 @@
 [is='flynt-navigation-footer'] {
-  .navigation {
-    padding: $component-spacing-mobile 0;
+  font-size: 0.75rem;
```

Comment:
`<small>`, `.pill` and `.button--small` are using `0.8rem`. That's why I would propose to use `0.8rem` here as well (or change the BaseStyle).